tasks/common_packages.yml

```diff
@@ -34,3 +34,11 @@
       - locales
       - nfs-client
       - net-tools
+      - tree
```

Comment:
Why? I've never seen anyone use it

Reply:
Please close this PR and put them directly in [stuvus_config](https://github.com/Mr-Pi/stuvus_config) after #67  is merged.